### PR TITLE
GitHub.ClassroomAssistant version 2.0.3

### PR DESCRIPTION
**Pull request by julianprester**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17834)

### CODE REVIEW

File: manifests/g/GitHub/ClassroomAssistant/2.0.3/GitHub.ClassroomAssistant.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: GitHub.ClassroomAssistant
+PackageVersion: 2.0.3
+Installers:
+- Architecture: neutral
+  InstallerType: exe
+  InstallerUrl: https://github.com/education/classroom-assistant/releases/download/v2.0.3/Classroom.Assistant-2.0.3.Setup.exe
+  InstallerSha256: 3F5806D492050CD30841D50DAB3C9C45B64866F7F80EB1D6ED98E86F9D0810F4
```
Comment (jedieaston) [inline manifests/g/GitHub/ClassroomAssistant/2.0.3/GitHub.ClassroomAssistant.installer.yaml:10]:
```suggestion
  InstallerSha256: 3F5806D492050CD30841D50DAB3C9C45B64866F7F80EB1D6ED98E86F9D0810F4
  InstallerSwitches:
    Silent: "-s"
    SilentWithProgress: "-s"
```

File: manifests/g/GitHub/ClassroomAssistant/2.0.3/GitHub.ClassroomAssistant.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: GitHub.ClassroomAssistant
+PackageVersion: 2.0.3
+PackageLocale: en-US
+Publisher: GitHub
```
Comment (jedieaston) [inline manifests/g/GitHub/ClassroomAssistant/2.0.3/GitHub.ClassroomAssistant.locale.en-US.yaml:7]:
```suggestion
Publisher: GitHub, Inc.
```

File: manifests/g/GitHub/ClassroomAssistant/2.0.3/GitHub.ClassroomAssistant.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: GitHub.ClassroomAssistant
+PackageVersion: 2.0.3
+PackageLocale: en-US
+Publisher: GitHub
+PackageName: Classroom Assistant
```
Comment (jedieaston) [inline manifests/g/GitHub/ClassroomAssistant/2.0.3/GitHub.ClassroomAssistant.locale.en-US.yaml:8]:
```suggestion
PackageName: classroom-assistant
```
To match the Add and Remove Programs table.

**Review by jedieaston — CHANGES_REQUESTED** [review]
It turns out that Classroom Assistant uses the same installer as Atom, so it has undocumented silent install arguments. I've added them, once you accept that and the changes to match the ARP table you should be good to go!

**Comment by julianprester** [comment]
Thank you @jedieaston I've committed the suggested changes.

**Review by jedieaston — APPROVED** [review]
:shipit:

**Pull request merged**